.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,64 @@
+name: Bug Report
+description: Found a Bug that needs to be fixed?
+body:
+- type: markdown
+  attributes:
+    value: |-
+      Please join the [Discord Server](https://discord.gg/0hMr4ce0tIl3SLv5) for questions or ask them in [our Discussions](https://github.com/DV8FromTheWorld/JDA/discussions).
+      
+      Keep in mind that this isn't the place to learn Java.  
+      Please head over to [StackOverflow](https://stackoverflow.com/questions/tagged/java) for your general programming questions.
+- type: checkboxes
+  attributes:
+    label: General Troubleshooting
+    description: You confirm to have made the following checks first.
+    options:
+      - label: I have checked for similar issues on the Issue-tracker.
+        required: true
+      - label: I have updated to the [latest JDA version](https://ci.dv8tion.net/job/JDA/lastSuccessfulBuild/)
+        required: true
+      - label: I have checked the branches or the maintainers' PRs for upcoming bug fixes.
+        required: true
+- type: textarea
+  attributes:
+    label: "Expected Behaviour"
+    description: "What did you expect JDA to do?"
+    placeholder: "JDA should do ..."
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: "Code Example for Reproduction Steps"
+    description: |-
+      Please add the code you use to reproduce this problem.  
+      Make sure to remove or replace any sensitive data like your Bot's token.
+      Leave this empty or put "N/A" if you don't have a reproducible setup.
+      
+      The provided text will be rendered as Java code, so you don't have to provide a Code block for it.
+    render: java
+    placeholder: |-
+      public void causeError() {
+          throw new Exception("Error!");
+      }
+- type: textarea
+  attributes:
+    label: "Code for JDABuilder or DefaultShardManagerBuilder used"
+    description: |-
+      Please provide the code used to create your JDA or ShardManager instance.  
+      Make sure to remove or replace any sensitive data like your Bot's token.
+      Leave this empty or put "N/A" if this isn't affected by the JDABuilder/DefaultShardManagerBuilder setup.
```

Comment:
I don't trust the issuer to actually know whether a bug is caused by their builder setup, lets always request this.

```suggestion
```

Reply:
It probably doesn't make sense for error since some bugs are simply "it doesn't run"